packages/sanity/src/core/comments/helpers.ts

```diff
@@ -10,6 +10,9 @@ export function useCommentHasChanged(message: CommentMessage): boolean {
   return useMemo(() => !isEqual(prevMessage, message), [prevMessage, message])
 }
 
+/**
+ * @internal
```

Comment:
I see a mix of using `@internal` + `@hidden` and just `@internal` in changed doc blocks, is that intentional? What is determining whether it should be `@hidden` if so?

Reply:
Ah, my bad. When I started this I was a bit confused on what was needed (there was a lot of errors to work through doing breath first export). Ill remove hidden when for these internals